README.md

```diff
@@ -85,6 +85,15 @@ Example manifest:
 }
 ```
 
+### input
+
+Type: `string`
+Default: `bundle.dest`
+Required: `false`
+
+The name of your filename intro used in creating manifest. 
```

Comment:
Change to something along the lines of: "The filename used as the input key in the generated manifest map."